mmv1/third_party/terraform/tests/resource_redis_instance_test.go

```diff
@@ -142,9 +141,8 @@ resource "google_redis_instance" "test" {
     maxmemory-policy       = "noeviction"
     notify-keyspace-events = ""
   }
-  <% unless version == 'ga' -%>
   transit_encryption_mode = "SERVER_AUTHENTICATION"
-  <% end -%>
+  redis_version = "REDIS_5_0"
```

Comment:
Does this test successfully update the resource rather than recreating it? It looks like it may still update the region which could trigger a recreation without actually calling the upgrade method

Reply:
Were you able to verify that this test updates the resource rather than recreating?

Reply:
yessir, I make it don't destroy so it's future proof